src/ExpressionEvaluator/Core/Source/FunctionResolver/VisualBasic/Scanner.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.Utilities;
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator
+{
+    internal sealed partial class MemberSignatureParser
```

Comment:
How much does this have in common with the C# scanner? Is there potential to factor out?